marketplace-core/pom.xml

```diff
@@ -97,6 +97,10 @@
       <groupId>org.webjars.bower</groupId>
       <artifactId>angular-ui-bootstrap-bower</artifactId>
     </dependency>
+    <dependency>
```

Comment:
@YuryBY I agree with @pamval that adding any dependency to kettle should always be something to be done after serious pondering. In this particular case the design is that marketplace-*core* *can* not depend on kettle. If it did, it wouldn't be *core*. 

I also prefer not to duplicate code when possible. Like @pamval mentioned, the HttpClientManager and HttpClientUtil seem like they can live in their own module that would be imported by both Kettle and Marketplace. I'm guessing that this http client management module will also be used in other parts of the pentaho suite.

The question remains what would be the best place to have the build for such artifact. To figure that out it is best to reach out to the build team or @pentaho/2-1b.

*As a last resort*, if having a common module is not possible for some reason, it is best to duplicate the code in marketplace rather than have a dependency on kettle.